ark/type/src/parser/tuple.ts

```diff
@@ -52,7 +59,8 @@ export const parseTupleLiteral = (def: List, ctx: ParseContext): TypeNode => {
 			elementDef = elementDef[1]
 			isVariadic = true
 		}
-		const value = ctx.scope.parse(elementDef, ctx)
+		const parsedData = parseKeyValueEntry(i, elementDef)
```

Comment:
Rename this also to something like `parsedEntry`- in the context of AT, data refers to user input. Anything else should have a more specific suffix or none at all.